### ISSUE TITLE
Set is_human_readable to false

### ISSUE THREAD

**Issue by dtolnay** [TyOverby/bincode#215]
I have not released this yet but I will in the next few days. https://github.com/serde-rs/serde/pull/1044

```rust
trait Serializer {
    /* ... */

    fn is_human_readable(&self) -> bool { true }
}

trait Deserializer<'de> {
    /* ... */

    fn is_human_readable(&self) -> bool { true }
}
```

Bincode should override these to false to get more compact representation for types like IpAddr and Datetime.

**Comment by TyOverby** [comment]
👍

**Comment by dtolnay** [comment]
- [`Serializer::is_human_readable`](https://docs.serde.rs/serde/trait.Serializer.html#method.is_human_readable)
- [`Deserializer::is_human_readable`](https://docs.serde.rs/serde/trait.Deserializer.html#method.is_human_readable)

**Comment by Marwes** [comment]
Would there be any reason to let this be configureable? (ala https://github.com/TyOverby/bincode/pull/214 ?). I suppose the only reason one would not want the human readable representation is to preserve backwards compatibility with the format of older versions.

**Comment by dtolnay** [comment]
I would not attempt to make one version of bincode parse every bincode data ever written. People can depend on both 0.9 and 1.0, deserialize with 0.9, serialize with 1.0 if necessary to migrate data.

**Comment by dtolnay** [comment]
Marking for 1.0 because doing this later would be a breaking change.

**Comment by TyOverby** [comment]
@Marwes: there's no reason that this couldn't be configurable, but I'm wondering if we should just add it and make it the default before bincode hits 1.0.0.

**Comment by RReverser** [comment]
So what's the further plan for this? Is this going to wait for 1.0 or is there a chance this fix could be released earlier in minor update?

**Comment by TyOverby** [comment]
waiting for 1.0, which ideally will be released early next week.

**Comment by RReverser** [comment]
@TyOverby Obviously this is taking more than a week, which is understandable and I'm fine with, but could you please publish intermediate version with this fix?

**Comment by TyOverby** [comment]
@RReverser: published!

**Comment by RReverser** [comment]
@TyOverby Thank you!

**Comment by RReverser** [comment]
Oh, you published actual 1.0.0 release? Awesome!

**Comment by TyOverby** [comment]
Just for you @RReverser! :D
